data/campaigns/Under_the_Burning_Suns/scenarios/09_Blood_is_Thicker_Than_Water.cfg

```diff
@@ -479,7 +479,7 @@
         [/message]
         [message]
             id=scout1
-            message= _ "Ha, stupid creature. But we’re in luck, the council was looking for the last of these spies."
+            message= _ "Ha, stupid creature. But we’re in luck, the Council was looking for the last of these spies."
```

Comment:
Here with council.

Reply:
My guess is the scout is referring to the Iron Council mentioned several times in this scenario, for example:
https://github.com/wesnoth/wesnoth/blob/6c78d376fe8f4d6b62d6392bef738e500854851d/data/campaigns/Under_the_Burning_Suns/scenarios/09_Blood_is_Thicker_Than_Water.cfg#L1208